controllers/admin/AdminImagesController.php

```diff
@@ -178,6 +192,51 @@ public function __construct()
                 ],
                 'submit' => ['title' => $this->trans('Save', [], 'Admin.Actions')],
             ],
+            'optimization' => [
+                'title' => $this->trans('Advanced optimization options', [], 'Admin.Design.Feature'),
+                'icon' => 'icon-picture',
+                'top' => '',
+                'bottom' => '',
+                'description' => $this->trans('Optimize bandwidth and disk usage.', [], 'Admin.Design.Help'),
+                'fields' => [
+                    'PS_IMAGEOPT_NO_ENLARGE' => [
+                        'type' => 'bool',
+                        'title' => $this->trans('Never create images larger than the base image', [], 'Admin.Design.Feature'),
+                        'required' => false,
+                        'is_bool' => true,
+                        'hint' => $this->trans('Never add a white or transparent border around an image.', [], 'Admin.Design.Help'),
```

Comment:
```suggestion
                        'hint' => $this->trans('Only add images without borders.', [], 'Admin.Design.Help'),
```

Is it important to specify white and transparent, or is it any border? also, I can't find this wording on the pictures, is it a help text or a label? The domain might change accordingly.

Reply:
It is a hint, i.e. the popup that shows when hovering over the option.
@Hlavtox 's suggestion is much clearer.

Reply:
`Only add images without borders.` does not make sense.

Thanks @lmeyer1 ;-)